app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -753,6 +754,17 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
                     !newCategory.isInsideRepository() -> renameCategory(oldCategory, CategoryRenameError.DestinationOutsideRepo)
                     else -> lifecycleScope.launch(Dispatchers.IO) {
                         moveFile(oldCategory.file, newCategory)
+
+                        //associate the new category with the last category's timestamp in history
+                        val preference = getSharedPreferences("recent_password_history", Context.MODE_PRIVATE)
+                        val timestamp = preference.getString(oldCategory.file.absolutePath)
```

Comment:
I don't know the exact charset supported by preference keys. @msfjarvis Do you know whether arbitrary paths could cause an issue here? Base64 or a hash might be safer.

Reply:
> I don't know the exact charset supported by preference keys. @msfjarvis Do you know whether arbitrary paths could cause an issue here? Base64 or a hash might be safer.

It's probably UTF_8 but a base64 hash is the safest bet.

Reply:
So... should I change the path to a hash...?

Reply:
Yes please

Reply:
Okay, i'll do it tomorrow :)

Reply:
I had some time so I did it, hope you don't mind.

Reply:
Great :)